### DISCUSSION TITLE
Welcome to aws-inventory Discussions!

### DISCUSSION THREAD

**Discussion by janiko71** [janiko71/aws-inventory#27] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jagadesh1982** [comment]
Thank you very much for the code. iam trying to enhance the code by adding delete  and gather details by tags ( filter tags) functionality. Can you suggest me on how to move forward.

**Reply by janiko71** [reply]
Hi, you're welcome. This script is not regularly maintained, AWS is adding services so frequently! Enhancements are always welcome, you can either fork the repo or pull requests, but I don't have much time theses days to look at them.

**Comment by jagadesh1982** [comment]
Thank you Very Much for replying back Jainko. As you said, will fork the code and try to enhance this. But can you help me on how can i get a inventory by passing tags. i see additional paramters are being passed but cant get the inventory by tags. can you guide me on this

**Comment by aomiglionni** [comment]
Is it possible to change output from json to csv?
